source/extensions/omni.isaac.lab/omni/isaac/lab/envs/mdp/events.py

```
@@ -195,6 +195,11 @@ def __init__(self, cfg: EventTermCfg, env: ManagerBasedEnv):
         texture_paths = cfg.params.get("texture_paths")
         event_name = cfg.params.get("event_name")
         texture_rotation = cfg.params.get("texture_rotation", (0.0, 0.0))
+        replicate_physics = cfg.params.get("replicate_physics")
+
+        # check to make sure replicate_physics is set to False, else raise warning
+        if replicate_physics:
+            raise Warning("replicate_physics is set to True, meaning all environments will have same textures applied.")
```

This is never desired. You should throw a runtime error and tell the user to set this parameter as false in the InteractiveSceneCfg

---

Do you mean we should change this to an error just in here? Or do you mean we should move the error to within the InteractiveSceneCfg?